frontend/src/components/Pages/InspectionPage/InspectionSection.tsx

```diff
@@ -60,26 +116,76 @@ export interface DeckMissionType {
     }
 }
 
-export interface OngoingMissionType {
+export interface DeckMissionCount {
+    [color: string]: {
+        count: number
+        message: string
+    }
+}
+
+export interface ScheduledMissionType {
     [missionId: string]: boolean
 }
 
+interface IInspectionProps {
+    refreshInterval: number
+    updateScheduledMissionsMap: (areaMissions: DeckMissionType) => Promise<void>
+    scheduledMissions: ScheduledMissionType
+    ongoingMissions: ScheduledMissionType
+}
+
+const getDeadlineInspection = (deadline: Date) => {
+    const deadlineDays = getDeadlineInDays(deadline)
+    switch (true) {
+        case deadlineDays <= 1:
+            return 'red'
+        case deadlineDays > 1 && deadlineDays <= 7:
+            return 'orange'
```

Comment:
```suggestion
            return 'red'
``?

Reply:
We don't really know this at this point, do we? But can move it to being red if in less than a week